test/integration/render-tests/geojson/fractional-minzoom/style.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "width": 512,
+      "height": 512
+    }
```

Comment:
Could we simplify this test a tad by:
 - width & height: 64
 - center 0,0
 - data: literal geojson point